src/html5_parser/parser/mod.rs

```diff
@@ -3458,10 +3458,10 @@ mod test {
         node_create!(parser, "div");
         node_create!(parser, "p");
         node_create!(parser, "button");
-        assert_eq!(parser.is_in_scope("p", Scope::Regular), true);
-        assert_eq!(parser.is_in_scope("p", Scope::Button), false);
-        assert_eq!(parser.is_in_scope("p", Scope::ListItem), true);
-        assert_eq!(parser.is_in_scope("p", Scope::Select), false);
+        assert!(parser.is_in_scope("p", Scope::Regular));
+        assert!(!parser.is_in_scope("p", Scope::Button));
+        assert!(parser.is_in_scope("p", Scope::ListItem));
+        assert!(!parser.is_in_scope("p", Scope::Select));
```

Comment:
These changes are from `cargo clippy --fix`.  I've added a `make fix` task to help with this.